apps/nestjs-backend/src/features/selection/selection.service.ts

```diff
@@ -796,4 +796,17 @@ export class SelectionService {
     await this.recordOpenApiService.deleteRecords(tableId, recordIds);
     return { ids: recordIds };
   }
+
+  async duplicate(tableId: string, rangesRo: IRangesRo) {
+    const { records } = await this.getSelectionCtxByRange(tableId, rangesRo);
+    const fields = records.map(({ fields }) => {
```

Comment:
Should ignore fields with `isComputed == true`

Reply:
```
const records: <{
    id: string;
    fields: Record<string, unknown>;
}>[]
```
The type of fields is `Record<string,unkown>` instead of IfieldInstance